pkg/ocibundle/sif/bundle_linux_test.go

```diff
@@ -65,12 +66,17 @@ func TestFromSif(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			tcBundlePath, err := ioutil.TempDir("", "bundle")
```

Comment:
as mentioned, bundle directory is created by `bundle.Create` if it doesn't exist, it is not a requirement explaining why it worked before

Reply:
removed all changes to `pkg/ocibundle/sif/*`